packages/components/src/components/input/input.jsx

```diff
@@ -28,6 +29,7 @@ const Input = (
         e.target.value = input_value;
         props.onChange(e);
     };
+    const dataLpignore = lpignore || (props.type === 'password' ? false : true);
```

Comment:
does false work, or do we need to make it undefined so it doesn't set the prop on the field at all?

Reply:
if a user pass `false` then we should check the field type (as we suppose to not show it for text/number types). also, I can validate it here to also trigger false.

Reply:
that's not what I meant, I mean, does setting `data-lpignore={false}` make lastpass icon show, or as long as it sees a `data-lpignore` prop on the field it will hide?

Reply:
do we need a `lpignore` prop for this or can we always set it to true except for type === 'password'?

Reply:
I think just doing this is sufficient `data-lpignore={props.type === 'password' ? undefined : true}`

Reply:
@negar-binary seems lastpass autmaticaly choose whether to show the icon or not, by considering input name, class, or Id. so somehow we cannot force it to show its' icon. `false/undefined` not work for it